helios-testing/src/test/java/com/spotify/helios/testing/DefaultDeployerTest.java

```diff
@@ -42,10 +41,9 @@
 import static java.util.Collections.emptyMap;
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
+import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.class)
```

Comment:
I've seen this in a few other test classes as well. Just for my own knowledge, What's this supposed to do and why don't we need it?